<!DOCTYPE html>
<!-- saved from url=(0035)http://ibisngs.github.io/knime4ngs/ -->
<html><head><meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
    <meta charset="utf-8">
    <meta http-equiv="X-UA-Compatible" content="chrome=1">
    <meta name="description" content="RSA : ">

    <!-- <link rel="stylesheet" type="text/css" media="screen" href="./Knime4NGS_files/stylesheet.css"><link rel="stylesheet" type="text/css" media="screen" href="./Knime4NGS_files/stylesheet.css"> -->

    <title>RSA</title>
	<style>
	/*******************************************************************************
Slate Theme for GitHub Pages
by Jason Costello, @jsncostello
*******************************************************************************/

@import url(github-light.css);

/*******************************************************************************
MeyerWeb Reset
*******************************************************************************/

html, body, div, span, applet, object, iframe,
h1, h2, h3, h4, h5, h6, p, blockquote, pre,
a, abbr, acronym, address, big, cite, code,
del, dfn, em, img, ins, kbd, q, s, samp,
small, strike, strong, sub, sup, tt, var,
b, u, i, center,
dl, dt, dd, ol, ul, li,
fieldset, form, label, legend,
table, caption, tbody, tfoot, thead, tr, th, td,
article, aside, canvas, details, embed,
figure, figcaption, footer, header, hgroup,
menu, nav, output, ruby, section, summary,
time, mark, audio, video {
  margin: 0;
  padding: 0;
  border: 0;
  font: inherit;
  vertical-align: baseline;
}

/* HTML5 display-role reset for older browsers */
article, aside, details, figcaption, figure,
footer, header, hgroup, menu, nav, section {
  display: block;
}

ol, ul {
  list-style: none;
}

table {
  border-collapse: collapse;
  border-spacing: 0;
}

/*******************************************************************************
Theme Styles
*******************************************************************************/

body {
  box-sizing: border-box;
  color:#373737;
  background: #212121;
  font-size: 16px;
  font-family: 'Myriad Pro', Calibri, Helvetica, Arial, sans-serif;
  line-height: 1.5;
  -webkit-font-smoothing: antialiased;
}

h1, h2, h3, h4, h5, h6 {
  margin: 10px 0;
  font-weight: 700;
  color:#222222;
  font-family: 'Lucida Grande', 'Calibri', Helvetica, Arial, sans-serif;
  letter-spacing: -1px;
}

h1 {
  font-size: 36px;
  font-weight: 700;
}

h2 {
  padding-bottom: 10px;
  font-size: 32px;
  background: url('../images/bg_hr.png') repeat-x bottom;
}

h3 {
  font-size: 24px;
}

h4 {
  font-size: 21px;
}

h5 {
  font-size: 18px;
}

h6 {
  font-size: 16px;
}

p {
  margin: 10px 0 15px 0;
}

footer p {
  color: #f2f2f2;
}

a {
  text-decoration: none;
  color: #007edf;
  text-shadow: none;

  transition: color 0.5s ease;
  transition: text-shadow 0.5s ease;
  -webkit-transition: color 0.5s ease;
  -webkit-transition: text-shadow 0.5s ease;
  -moz-transition: color 0.5s ease;
  -moz-transition: text-shadow 0.5s ease;
  -o-transition: color 0.5s ease;
  -o-transition: text-shadow 0.5s ease;
  -ms-transition: color 0.5s ease;
  -ms-transition: text-shadow 0.5s ease;
}

a:hover, a:focus {text-decoration: underline;}

footer a {
  color: #F2F2F2;
  text-decoration: underline;
}

em {
  font-style: italic;
}

strong {
  font-weight: bold;
}

img {
  position: relative;
  margin: 0 auto;
  max-width: 739px;
  padding: 5px;
  margin: 10px 0 10px 0;
  border: 1px solid #ebebeb;

  box-shadow: 0 0 5px #ebebeb;
  -webkit-box-shadow: 0 0 5px #ebebeb;
  -moz-box-shadow: 0 0 5px #ebebeb;
  -o-box-shadow: 0 0 5px #ebebeb;
  -ms-box-shadow: 0 0 5px #ebebeb;
}

p img {
  display: inline;
  margin: 0;
  padding: 0;
  vertical-align: middle;
  text-align: center;
  border: none;
}

pre, code {
  width: 100%;
  color: #222;
  background-color: #fff;

  font-family: Monaco, "Bitstream Vera Sans Mono", "Lucida Console", Terminal, monospace;
  font-size: 14px;

  border-radius: 2px;
  -moz-border-radius: 2px;
  -webkit-border-radius: 2px;
}

pre {
  width: 100%;
  padding: 10px;
  box-shadow: 0 0 10px rgba(0,0,0,.1);
  overflow: auto;
}

code {
  padding: 3px;
  margin: 0 3px;
  box-shadow: 0 0 10px rgba(0,0,0,.1);
}

pre code {
  display: block;
  box-shadow: none;
}

blockquote {
  color: #666;
  margin-bottom: 20px;
  padding: 0 0 0 20px;
  border-left: 3px solid #bbb;
}


ul, ol, dl {
  margin-bottom: 15px
}

ul {
  list-style-position: inside;
  list-style: disc;
  padding-left: 20px;
}

ol {
  list-style-position: inside;
  list-style: decimal;
  padding-left: 20px;
}

dl dt {
  font-weight: bold;
}

dl dd {
  padding-left: 20px;
  font-style: italic;
}

dl p {
  padding-left: 20px;
  font-style: italic;
}

hr {
  height: 1px;
  margin-bottom: 5px;
  border: none;
  background: url('../images/bg_hr.png') repeat-x center;
}

table {
  border: 1px solid #373737;
  margin-bottom: 20px;
  text-align: left;
 }

th {
  font-family: 'Lucida Grande', 'Helvetica Neue', Helvetica, Arial, sans-serif;
  padding: 10px;
  background: #373737;
  color: #fff;
 }

td {
  padding: 10px;
  border: 1px solid #373737;
 }

form {
  background: #f2f2f2;
  padding: 20px;
}

/*******************************************************************************
Full-Width Styles
*******************************************************************************/

.outer {
  width: 100%;
}

.inner {
  position: relative;
  max-width: 640px;
  padding: 20px 10px;
  margin: 0 auto;
}

#forkme_banner {
  display: block;
  position: absolute;
  top:0;
  right: 10px;
  z-index: 10;
  padding: 10px 50px 10px 10px;
  color: #fff;
  background: url('../images/blacktocat.png') #0090ff no-repeat 95% 50%;
  font-weight: 700;
  box-shadow: 0 0 10px rgba(0,0,0,.5);
  border-bottom-left-radius: 2px;
  border-bottom-right-radius: 2px;
}

#header_wrap {
  background: #212121;
  background: -moz-linear-gradient(top, #373737, #212121);
  background: -webkit-linear-gradient(top, #373737, #212121);
  background: -ms-linear-gradient(top, #373737, #212121);
  background: -o-linear-gradient(top, #373737, #212121);
  background: linear-gradient(top, #373737, #212121);
}

#header_wrap .inner {
  padding: 50px 10px 30px 10px;
}

#project_title {
  margin: 0;
  color: #fff;
  font-size: 42px;
  font-weight: 700;
  text-shadow: #111 0px 0px 10px;
}

#project_tagline {
  color: #fff;
  font-size: 24px;
  font-weight: 300;
  background: none;
  text-shadow: #111 0px 0px 10px;
}

#downloads {
  position: absolute;
  width: 210px;
  z-index: 10;
  bottom: -40px;
  right: 0;
  height: 70px;
  /*background: url('../images/icon_download.png') no-repeat 0% 90%;*/
  background: none;
}

.zip_download_link {
  display: block;
  float: right;
  width: 90px;
  height:70px;
  text-indent: -5000px;
  overflow: hidden;
  background: url(../images/sprite_download.png) no-repeat bottom left;
  }

.tar_download_link {
  display: block;
  float: right;
  width: 90px;
  height:70px;
  text-indent: -5000px;
  overflow: hidden;
  background: url(../images/sprite_download.png) no-repeat bottom right;
  margin-left: 10px;
}

.zip_download_link:hover {
  background: url(../images/sprite_download.png) no-repeat top left;
}

.tar_download_link:hover {
  background: url(../images/sprite_download.png) no-repeat top right;
}

#main_content_wrap {
  background: #f2f2f2;
  border-top: 1px solid #111;
  border-bottom: 1px solid #111;
}

#main_content {
  padding-top: 40px;
}

#footer_wrap {
  background: #212121;
}



/*******************************************************************************
Small Device Styles
*******************************************************************************/

@media screen and (max-width: 480px) {
  body {
    font-size:14px;
  }

  #downloads {
    display: none;
  }

  .inner {
    min-width: 320px;
    max-width: 480px;
  }

  #project_title {
  font-size: 32px;
  }

  h1 {
    font-size: 28px;
  }

  h2 {
    font-size: 24px;
  }

  h3 {
    font-size: 21px;
  }

  h4 {
    font-size: 18px;
  }

  h5 {
    font-size: 14px;
  }

  h6 {
    font-size: 12px;
  }

  code, pre {
    min-width: 320px;
    max-width: 480px;
    font-size: 11px;
  }

}

	</style>
  </head>

  <body>

    <!-- HEADER -->
    <div id="header_wrap" class="outer">
        <header class="inner">
         <!-- <a id="forkme_banner" href="https://github.com/ibisngs/knime4ngs">View on GitHub</a> -->

          <h1 id="project_title">RSA</h1>
          <h2 id="project_tagline"></h2>

            <section id="downloads">
            <!--  <a class="zip_download_link" href="https://github.com/ibisngs/knime4ngs/raw/master/resource-bundle.tar.gz">Download this project as a .zip file</a> -->
              <!--<a class="tar_download_link" href="https://github.com/ibisngs/knime4ngs/tarball/master">Download this project as a tar.gz file</a> -->
            </section>
        </header>
    </div>

    <!-- MAIN CONTENT -->
    <div id="main_content_wrap" class="outer">
      <section id="main_content" class="inner">
	<p>The basic purpose of RSA is to reveal differences among genes and transcripts from different states of the same species using a simple approach that does not require familiarity with Linux and algorithm implementation. 

	
	</p><h2 id="QuickStart">Quick Start</h2>
		<section>
			<h3 class="nodent">Prerequisites</h3>
				<ul type="square">
    					<li>Linux OS (required)</li>				 
					
				</ul>

				
			<h3 class="nodent">Download</h3>
				<p>You can download and install the RSA in &nbsp;<a href="https://pan.baidu.com/s/1c11Es5E">RSA</a></p>
				
				
				
			<h3 class="nodent">Flow Chart</h3>
				All steps from raw read quality control to finding out differences among genes and transcripts. <br>
				<img src="https://github.com/lachanglalala/RSA.github.io/blob/master/122.jpg?raw=true"></img>
				

			<h3 class="nodent">Install </h3>
			<pre>$ RSA install.sh </pre>
				<p>In software folder including some software that we have been tested. It including fastqc, Trimmomatic, Hisat2，samtools, stringtie, R R packages PLEK and TransDecoder According to the download link all software can be download on the internet. </p>
1. FastQC software (http://www.softpedia.com/get/Science-CAD/FastQC.shtml, version 0.11.3 or later).
2. Trimmomatic software (http://www.usadellab.org/cms/?page=trimmomatic, version 0.33 or later).
3. HISAT2 software  (http://ccb.jhu.edu/software/hisat2; version 2.0.1 or later).
4. StringTie software (http://ccb.jhu.edu/software/stringtie, version 1.22 or later). 
5. R (https://www.r-project.org, version 3.2.2 or later). Some R package 
6. R package 
$install.packages("devtools",repos="http://cran.us.r-project.org")
$source("http://www.bioconductor.org/biocLite.R")
biocLite(c("alyssafrazee/RSkittleBrewer","ballgown", "genefilter","dplyr","devtools"))
$install.packages("pheatmap")
$install.packages("ggplot2")
7. TransDecoder http://transdecoder.github.io/
8. Samtools http://www.htslib.org/
9.Python https://www.python.org/
10 PLEK https://sourceforge.net/projects/plek/

			<h3 class="nodent">Configuration</h3>
				RSA will install all the software and configure the PATH. We unified installed in /opt/biosoft/ folder below. This step include set up PATH, but you can set install folder and configuration PATH according to your habits. Install FastQC in /opt/biosoft/ or you can modify the running script
	  
	  <h3 class="nodent">Run</h3>
				<p>First Running RSAprepare </p>
<pre><p>$sh RSAprepare</p></pre>
<p>It will make a “welcome” work folder, which contains two folders named “control” and “experiment”. Put two groups FASTQ data which need to be analyze in “control” folder and “experiment” folder respectively. Put genome sequence files and genome annotation file in “welcome” folder. </p>
<p>
Make sure put all the files in respectively folders. If your data is single-ended sequencing choose RSA_single_lncRNA or RSA_single_mRNA. If your data is pair-ended sequencing choose RSA_pair_end_lncRNA or RSA_pair_end_mRNA.<br>
<pre>
$sh RSA_single_lncRNA<br>
$sh RSA_single_mRNA<br>
$sh RSA_pair_end_lncRNA<br>
$sh RSA_pair_end_mRNA</p></pre>
      </section>
	  <h3 class="nodent">RESULT</h3>
				RSA will install all the software and configure the PATH. We unified installed in /opt/biosoft/ folder below. This step include set up PATH, but you can set install folder and configuration PATH according to your habits. Install FastQC in /opt/biosoft/ or you can modify the running script 
				<h3 class="nodent">Source Code</h3>
				The complete source code of RSA is get by contacting email <a>liuxu1658@163.com</a>
	<h2 id="FAQs">FAQs</h2>
	      	<section>
			<ol type="1">
				<li> 
					<p><strong>Why run fail?</strong></p>
					<p>The software version of RSA software folder may different with you download. When the installation of the PATH variable is not set successfully you can modify the running script inside the call PATH address resolved.</p>
				</li>
				<li>
					<p><strong>How long does it take to run?</strong></p>
					<p>It depends on your computer configuration and the amount of data.</p>
				</li>
				<li>
					<p><strong>How high is the computer configuration?</strong></p>
					<p>Recommended hardware configuration (64-bit computer running either Linux or Mac OS X (10.4 Tiger or later); 4 GB of RAM (16 GB preferred). </p>
				</li>
				<li> 
					<p><strong>What is the advantage of this software?</strong></p>
					<p>RSA can substantially lower the effort required for scientists entering into the areas of RNA-Seq and Big Data, especially for those not familiar with Linux.</p>
				</li>
				
			</ol>
                </section>


	<h2 class="nodent">Contact</h2>
	<section>
		<a>liuxu1658@163.com</a><br>
<a>ChengRui5603@163.com</a>
	</section>
    </section></div>

    <!-- FOOTER  -->
    <div id="footer_wrap" class="outer">
      <footer class="inner">
        <!-- <p class="copyright">RSA maintained by <a href="https://github.com/ibisngs">ibisngs</a></p> -->
        <!-- <p>Published with <a href="https://pages.github.com/">GitHub Pages</a></p><p class="copyright">RSA maintained by <a href="https://github.com/ibisngs">ibisngs</a></p> -->
        <!-- <p>Published with <a href="https://pages.github.com/">GitHub Pages</a></p> -->
      </footer>
    </div>
  

</body></html>
